db/core/models/__init__.py

```diff
@@ -26,6 +26,68 @@
 from .utils import AsyncBaseModel, BaseDiscordMemberWrapper, DiscordMember
 
 
+class Action(BaseDiscordMemberWrapper):
```

Comment:
Do you still want it to be called committee action? granted non-committee being actioned won't be common but is possible and other servers than CSS may wish to use actions in different ways

Reply:
I would say yes because although they can be assigned *to* non committee members they should only be able to be *created* by committee members. Other servers might wish a different way but I would say that is out of scope for this feature. It's not a complete member's individual todo list system. It's a community group's committee actions tracker, thus has to be dependant upon the committee role assigning them.